### ISSUE TITLE
[Feature Request]: True Fullscreen Preview

### ISSUE THREAD

**Issue by bobcate** [anapnoe/stable-diffusion-webui-ux#208]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Like with the original UI, clicking on a generated image overlays that image on the screen, fully covering the screen space.

### Additional information

Something similar is mentioned in #5. Right now, it's working like this:

![1](https://github.com/anapnoe/stable-diffusion-webui-ux/assets/34423403/6d762b66-004c-471e-a5e3-d42907d98747)
_The generated images are listed in the middle of the screen._

![2](https://github.com/anapnoe/stable-diffusion-webui-ux/assets/34423403/07140264-e075-4d69-9794-8a44016764c8)
_When we click on one, it supposedly goes to fullscreen mode._

But this is not truly a fullscreen view. It just fills the designated panel.
This functionality is good. So I'd like to suggest:
- Clicking on the image again opens an overlay with the image, fully covering the screen,
or
- Adding a button to the top right of the image, next to the download and close buttons, which opens a fullscreen overlay.

#201 should also be taken into consideration if you decide to implement my suggestion as I also would like to see that request come to life. Maybe that one can be done inside the fullscreen overlay.

**Comment by ryankhart** [comment]
Agreed. My workaround for now is right-clicking on the image and opening the image in a new browser tab, but that's one more click that I want to do.
